core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -77,7 +77,8 @@ private UserPreferences(){}
     // Network
     private static final String PREF_ENQUEUE_DOWNLOADED = "prefEnqueueDownloaded";
     public static final String PREF_UPDATE_INTERVAL = "prefAutoUpdateIntervall";
-    private static final String PREF_MOBILE_UPDATE = "prefMobileUpdate";
+    public static final String PREF_MOBILE_UPDATE_OLD = "prefMobileUpdate";
```

Comment:
we can never get rid of this...

Reply:
Move as hard-coded string to the upgrade class? It is only used in one location